### PR TITLE
DomainTools Iris: Documentation - Add contributors

### PR DESCRIPTION
**Pull request by wesleya**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [x] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [x] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- We are just adding contributors to `domaintools_iris.json` to test the new partner guide for SOAR connectors
- Fix existing failed automated checks while we're at it so we can merge

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

File: domaintools_iris.json [hunk 1]
```diff
@@ -12,10 +12,14 @@
     "product_vendor": "DomainTools",
     "product_name": "DomainTools Iris",
     "product_version_regex": ".*",
-    "min_phantom_version": "5.0.0",
+    "min_phantom_version": "5.1.0",
```
Comment (wesleya) [inline domaintools_iris.json:15]: Fix "Min Phantom version in app json is too low." automated check

File: domaintools_iris_connector.py [hunk 2]
```diff
@@ -131,7 +127,8 @@ def _parse_response(self, action_result, r, response_json):
 
         if status == 503:
             error_message = error.get('message',
-                                      "There was an error processing your request. Please try again or contact support (http://www.domaintools.com/support) with questions.")
+                                      ("There was an error processing your request. "
+                                       "Please try again or contact support (http://www.domaintools.com/support) with questions."))
```
Comment (wesleya) [inline domaintools_iris_connector.py:131]: fix flake8 automated checks

File: domaintools_iris_connector.py [hunk 3]
```diff
@@ -312,23 +309,17 @@ def handle_action(self, param):
...
-            ret_val = self._pivot_action(param)
+            ret_val = self._reverse_whois_email(param)
         elif action_id == self.ACTION_ID_REVERSE_DOMAIN:
-            ret_val = self._reverse_domain(param)
+            ret_val = self._reverse_lookup_domain(param)
         elif action_id == self.ACTION_ID_LOAD_HASH:
-            data = {'pivot_type': 'search_hash', 'query_value': param['hash'], 'hash': param['hash']}
-            param.update(data)
-            ret_val = self._pivot_action(param)
+            ret_val = self._load_hash(param)
```
Comment (wesleya) [inline domaintools_iris_connector.py:318]: Could not find an implementation for reverse_lookup_domain, reverse_lookup_ip, load_hash, reverse_whois_email under an expected name

**Comment by wesleya** [comment]
This latest set of changes just get our code to pass the automated checks for Splunk Soar.

In addition to code review can you check a few things for me:
1. are you able to push a branch directly to this upstream repo?
2. are you able to approve this MR?
3. are you able to merge this MR?

They docs they sent us made it sound like we should be able to push our branch directly, then approve and merge to release ourselves. But I don't think I have permissions maybe? I opened this PR from our fork for now just to get fixed up for the auto checks. Wanted a quick sanity check before I ping someone for access.

**Review by ChuckWoodraska — APPROVED** [review]

**Pull request merged**